## PS3 Discussion: Randomization and Experiments

Let's continue to use the anchor data for today's coding practice. Students guessed the maximum speed of a house cat in miles per hour. 

In one condition, the "high" condition, students were prompted with the message 

"Is the maximum speed of an average house cat FASTER or SLOWER than **40** miles per hour?"

In the other condition, the "low" condition, students were prompted with

"Is the maximum speed of an average house cat FASTER or SLOWER than **3** miles per hour?"

The student then had to provide an estimate for the speed of a house cat. We want to learn: did the anchoring have an effect on students' estimates? Let's assume that students were randomly assigned to each group.

-------

**Data Dictionary/Codebook**

`cat`: estimated speed in mph

`before`: 1 = student knew the exercise beforehand, 0 = student did not know

`live.cats`: 1 = student lives with cats, 0 = student does not live with cats

`cond`: 1 = student was exposed to the "high" condition (40 mph), 0 = student was exposed to the "low" condition (3 mph)

In [1]:
# Run this cell to read in the data.
anchor <- read.csv("anchoring_data_FA20.csv")
head(anchor)

cat before live.cats cond
1  2  0      0         0   
2  1  0      0         1   
3 30  0      0         1   
4 25  0      0         1   
5 17  0      0         0   
6 50  0      0         1

Let's then "clean" the dataset as we did in last week's section. We want to remove some potentially problematic variables; we do NOT want to study people who (1) know about the goal of the experiment or (2) have lived with a cat, since they'll be more knowledgeable and have different answers.

**Quick Check 1**

Subset the data so we only get rows in which (1) the participant has NOT lived with a cat, and (2) the participant was NOT aware of the experiment/exercise. Save this to the variable `focus_anchor`.

In [2]:
# Can you try one-liner to generate the new variable focus_anchor?

focus_anchor <- subset(anchor, before==0 & live.cats==0)
head(focus_anchor)

cat before live.cats cond
1  2  0      0         0   
2  1  0      0         1   
3 30  0      0         1   
4 25  0      0         1   
5 17  0      0         0   
6 50  0      0         1

**Quick Check 2**

What is the treatment effect of being assigned to the high-group among the subset of participants that we just obtained?


In [5]:
library(estimatr) 
# estimatr is an R package providing a range of commonly-used linear estimators. Just run this line of code, do not change anything

difference_in_means(cat ~ cond, anchor, condition1 = '0', condition2 = '1') 
# What codes are needed in the brackets? Talk to your neighbors for 5min and we'll share results in class.
# You may refer to the R cheatsheet on bcourses

Design:  Standard 
     Estimate Std. Error  t value     Pr(>|t|) CI Lower CI Upper       DF
cond 18.75946   2.757204 6.803798 8.612699e-10 13.28673 24.23218 96.38872

**Quick Check 3**

Interpret the estimate

+17.48 is the treatment effect being assigned to high group relative to the low group on estimated cat speed

**Quick Check 4**

What is the standard error here?

2.757204

**Quick Check 5**

Interpret the standard error

The standard deviation of the sampling distribution of the treatment effect (i.e. difference in guesses of cat speed) is 2.455